File: python/amazonpricing.py
import requests

task_params = {
    'target': 'amazon_pricing',
    'query': 'B09H74FXNW',
    'parse': True,
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scraper-api.smartproxy.com/v2/scrape',
    json = task_params,
    auth = (username, password)
)

print(response.text)

File: nodejs/amazonpricing.js
const sdk = require('api')('@smartproxy/v1.0#25e7913l1ow524w');

sdk.auth('SPusername', 'SPpassword');
sdk.realTimeExample({target: 'amazon_pricing', query: 'B09H74FXNW', parse: true})
  .then(res => console.log(res))
  .catch(err => console.error(err));